bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/PerformanceLifecycleCallbacks.kt

```diff
@@ -206,6 +206,8 @@ class PerformanceLifecycleCallbacks internal constructor(
                 spanFactory.createCustomSpan(spanNameForPhase(activity,phase),
                     SpanOptions.DEFAULTS.within(viewLoadSpan)).apply {
                     setAttribute("bugsnag.span.category", "view_load_phase")
+                    setAttribute("bugsnag.view.name", "${activity::class.java.simpleName}")
+                    setAttribute("bugsnag.phase", "Activity${phase.spanName}")
```

Comment:
I think it would be better if we got this string directly from the `ViewLoadPhase` instead of deriving it here.

```suggestion
                    setAttribute("bugsnag.phase", phase.phaseName)
```